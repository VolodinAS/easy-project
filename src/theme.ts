const theme = {
    color: {
        typeTask: '#F8BFBF',
        typeFeature: '#B7CFFF',
        typeBug: '#FEFBBD',
        black: '#000000',
        lightPurple: '#CFC7D9',
        purple: '#7A52AA',
        lightGray: '#F5F5F5',
        darkGray: '#9C9C9C',
        red: '#EB0C0C',
        white: '#FFFFFF',
        smallText: '#999999',
        opacityGray: '#ACACAC',
        landingColors: {
            mainGray: '#D5E6EA',
            mainBlue: '#3848A9',
            benefitsItemBackground: '#EDD8BE',
            buttonStartBackground: '#E43958',
            footerBackground: '#343F53',
            headerLogo: '#010101',
            headerBackground: '#F1E7DA'
        },
        dashboardColors: {
            overlayButtonCreate: '#ABE0E7',
            overlayButtonYes: '#B4E7AB',
            overlayButtonNo: '#F9CCBE',
        }
    }
};

export default theme;
