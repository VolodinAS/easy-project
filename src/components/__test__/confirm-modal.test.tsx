/* eslint-disable check-file/folder-naming-convention */
import React from 'react';
import { describe, it, expect, jest } from '@jest/globals';
import { render, fireEvent } from '@testing-library/react';
import { ThemeProvider } from '@emotion/react';

import ConfirmModal from '../confirm-modal';
import theme from '../../theme';

describe('confirmModal', () => {
    it('render', () => {
        const onClose = jest.fn();
        const onConfirm = jest.fn();
        render(
            <ThemeProvider theme={theme}>
                <ConfirmModal onClose={onClose} onConfirm={onConfirm} />
            </ThemeProvider>    
        );
        expect(document.body).toMatchSnapshot();
    });
});