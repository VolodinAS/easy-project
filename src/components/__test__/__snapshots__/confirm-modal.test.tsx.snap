// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`confirmModal render 1`] = `
<body>
  <div>
    <div
      class="css-q7ennu"
      id="overlay"
    >
      <div
        class="css-fslfiw"
      >
        <div
          class="css-9rseyg"
        >
          <button
            class="css-4jvx38"
          >
            <img
              alt="Закрыть"
              src="test-file-stub"
            />
          </button>
        </div>
        <div
          class="css-19jvvxe"
        >
          <p
            class="css-ovqjzh"
          />
        </div>
        <div
          class="css-106mbhu"
        >
          <button
            class="css-a5dza1"
          >
            Нет
          </button>
          <button
            class="css-a5dza1"
            data-testid="button-delete"
          >
            Да
          </button>
        </div>
      </div>
    </div>
  </div>
</body>
`;
